src/openalea/strawberry/analysis.py

```diff
@@ -599,6 +609,12 @@ def mean_leaf_area(vid,g):
 def complete(vid, g):
     return g.property("complete").get(vid, False)
 
+
+def stage(vid, g):
+    _stage = g.property('Stade')
+    return next(iter(list(_stage[cid] for cid in g.components(vid) if cid in _stage)), None)
```

Comment:
Do not get it.
Why not just take the first element?
You do not need to convert  it to a list.
And you can use components_iter